dbt/adapters/bigquery/impl.py

```diff
@@ -907,3 +909,26 @@ def python_submission_helpers(self) -> Dict[str, Type[PythonJobHelper]]:
             "cluster": ClusterDataprocHelper,
             "serverless": ServerlessDataProcHelper,
         }
+
+    @classmethod
+    def render_column_constraint(cls, constraint: ColumnLevelConstraint) -> str:
+        if constraint.type == ConstraintType.not_null:
+            return super().render_column_constraint(constraint)
+        elif constraint.type == ConstraintType.primary_key:
+            return "primary key not enforced"
+        elif constraint.type == ConstraintType.foreign_key:
+            c = super().render_column_constraint(constraint)
+            return f"{c} not enforced"
```

Comment:
They seem like two distinct semantics; one where you call that constraint and another for which you don't. From an observer's point of view, it feels like there should be separate debugging messages of these two things don't get completed. You are being overcautious?